docs/Makefile

```diff
@@ -1,16 +1,18 @@
 PREFIX ?= /usr/local
 DATADIR ?= ${PREFIX}/share
 MANDIR ?= $(DATADIR)/man
-MANDOWN ?= mandown
+GO ?= go
+GOBIN ?= $(GOPATH)/bin
```

Comment:
```suggestion
GOBIN ?= $($(GO) env GOPATH)/bin
```

Reply:
I've dropped the setting of `$GOBIN` as per https://github.com/containers/netavark/pull/771/files#r1288638343.